docs/smart-contracts/data-types/primitive-data-types.md

```diff
@@ -173,11 +174,29 @@ For smart contract accounts, the string starts with "KT1".
 
 The next part of the string is a `Base58` encoded hash, followed by a 4-byte checksum.
 
+## Unit {#unit}
+
+In Tezos, the `unit` type represents no value.
```

Comment:
This could be confused with the never type. There is a value of type unit but there is only one.

I propose:
> In Tezos, a value of type `unit` holds no information.

Reply:
In that case I'll move it to the topic on complex types because it's really a variant that contains no info, right? I can handle it like the LIGO docs:
https://ligolang.org/docs/variants/unit/?lang=jsligo

Reply:
Changed the description to:

> In Tezos, the `unit` type is a special case of the variant type: a single value that holds no information. It is a placeholder where a variable is required but no other information is needed.

Reply:
- A type is not a value.
- The concept of variant type is more complex than the concept of the unit type so defining the unit type as a special case of a variant type is does not very helpful, especially if the page does not define variant types.

So, new proposition:

> In Tezos, the `unit` type is a type containing a single value. It is used as a placeholder where some type is required but no information is transmitted. It is the input type of functions taking no input, the output type of functions producing no output, and the storage type of contracts storing no information.

Reply:
Sorry, I did not realize that this was moved to a page talking about record and variants.

So if you want to mention the relationship with variants and records, the `unit` type is both a variant with a single constructor and a record with no field. But I'm still not convinced that this is worth mentioning.